internal/otlp/unmarshaler.go

```diff
@@ -18,17 +18,32 @@ import (
 	"go.opentelemetry.io/collector/internal/model"
 )
 
-// NewJSONTracesUnmarshaler returns a model.TracesUnmarshaler to decode from OTLP json bytes.
+// NewJSONTracesUnmarshaler returns a model.TracesUnmarshaler. Unmarshalls from OTLP json bytes.
```

Comment:
```suggestion
// NewJSONTracesUnmarshaler returns a model.TracesUnmarshaler. Unmarshals from OTLP json bytes.
```

I think it's just 1 l (see https://golang.org/pkg/encoding/json/) and Google returns more results for Unmarshals.

Reply:
Done.